install.sh

```diff
@@ -395,12 +398,21 @@ EOF
 
 exit_if_pkg_config_pc_file_missing "caffeine"
 
+compiler_version=$($FPM_FC --version)
+if [[ $compiler_version == *llvm* ]]; then
+  compiler_flag="-mmlir -allow-assumed-rank -g -Ofast"
```

Comment:
With the latest versions of flang, this is not necessary. However, maybe we should keep it in for now until I verify that the container of flang isn't a version that we need these flags for. The container was updated 7 days ago, so likely these flags aren't necessary for the CI version of flang.